18.0 js framework onboarding SAWER

sawer-odoo opened the pull request.

sawer-odoo committed:
[ADD] Chapter 1: Owl components - sawer

robodoo commented:
[![Pull request status dashboard](https://mergebot.odoo.com/odoo/tutorials/pull/1006.png)](https://mergebot.odoo.com/odoo/tutorials/pull/1006)

sawer-odoo committed:
[ADD] Chapter 2: Build a dashboard - points 1 to 4

[ADD] Chapter 2: Build a dashboard - points 5 to 11

sawer-odoo committed:
[ADD] Chapter 1: Build a Clicker game - points 1 to 10

[ADD] Chapter 1: Build a Clicker game - points 11 to 21

sawer-odoo committed:
[ADD] Chapter 2: Create a gallery view - point 1

[ADD] Chapter 2: Create a galler view - points 2 to 14

[FIX] runbot correction in validation.py

[FIX] real fix in validation.py

sawer-odoo committed:
[ADD] Chapter 3: Customize a kanban view

[IMP] little change in the way in change tho pager offset
